plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java

```diff
@@ -45,10 +59,27 @@ public TestingOracleServer()
     {
         super("wnameless/oracle-xe-11g-r2");
 
+        Failsafe.with(CONTAINER_RETRY_POLICY).run(this::createContainer);
+    }
+
+    private void createContainer()
+            throws Exception
+    {
         withCopyFileToContainer(MountableFile.forClasspathResource("init.sql"), "/docker-entrypoint-initdb.d/init.sql");
 
-        start();
+        Closeable cleanup = startOrReuse(this);
```

Comment:
Was your intention to both: add retries and make this a reusable container?
I'd suggest keeping these separate (two commits, or two PRs)

Reply:
It's not my intention. Let me use `start()` method instead.